### PR TITLE
Made heaven option on run away.

### PR DESCRIPTION
**Pull request by js1999**
Please review this, @casey-collab

### CODE REVIEW

File: english/coffee/nude-run/cold-water/run-away/run-away.md [hunk 1]
```diff
@@ -5,4 +5,6 @@ end, it doesn't even matter. You had to fall
 to lose it all, but in the end, it doesn't even
 matter.
 
-Because that badger s-o-b outruns you and kills you.
\ No newline at end of file
+Because that badger s-o-b outruns you and kills you.
+
+["Go to heaven"](../cold-water/run-away/heaven/heaven.md)
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/run-away/run-away.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/coffee/nude-run/cold-water/run-away/heaven/heaven.md [hunk 2]
```diff
@@ -0,0 +1 @@
+You float up to heaven and see the pearly gates.
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/run-away/heaven/heaven.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by js1999** [comment]
Please review this, @casey-collab

File: english/coffee/nude-run/cold-water/run-away/run-away.md [hunk 3]
```diff
@@ -5,4 +5,6 @@ end, it doesn't even matter. You had to fall
 to lose it all, but in the end, it doesn't even
 matter.
 
-Because that badger s-o-b outruns you and kills you.
\ No newline at end of file
+Because that badger s-o-b outruns you and kills you.
+
+["Go to heaven"](..run-away/heaven/heaven.md)
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/run-away/run-away.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/coffee/nude-run/cold-water/run-away/heaven/heaven.md (hunk 2 above)
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/run-away/heaven/heaven.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by js1999** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :lollipop:

**Pull request merged**